guacamole-docker/bin/start.sh

```diff
@@ -567,6 +567,38 @@ END
     ln -s /opt/guacamole/duo/guacamole-auth-*.jar   "$GUACAMOLE_EXT"
 }
 
+##
+## Adds properties to guacamole.properties witch configure the Cas
```

Comment:
Cas -> CAS

Reply:
as above

I found another 'Cas' in my code, on line 739.
 I'll change it too


Sorry